extensions/amp-ad-network-adsense-impl/0.1/test/test-amp-ad-network-adsense-impl.js

```diff
@@ -395,6 +395,9 @@ describes.sandboxed('amp-ad-network-adsense-impl', {}, () => {
       return createIframePromise().then(fixture => {
```

Comment:
Is there a test where height/width are auto?  With experiment on?  Without it on?